test/parallel/test-internal-fs-syncwritestream.js

```diff
@@ -41,6 +41,19 @@ const filename = path.join(common.tmpDir, 'sync-write-stream.txt');
   assert.strictEqual(fs.readFileSync(filename).equals(chunk), true);
 }
 
+// Throws if data is not of type Buffer.
+{
+  const stream = new SyncWriteStream(1);
```

Comment:
You can just get the stream using `new fs.WriteStream`,  or `fs.createWriteStream`, `SyncWriteStream` is being deprecated. That way this test should be in `test-fs-write-stream` instead.

Reply:
Thanks! I made the changes.

I debugged the file to see why I was getting the error that the `code` was expected `ERR_INVALID_ARG_TYPE` but actual was `undefined`.

It looks like require(`fs`) is grabbing my installed 8.x `fs` and not `lib/fs.js`. Thoughts on this?